docusaurus/docs/reactnative/contexts/message_input_context.mdx

```diff
@@ -5,7 +5,6 @@ title: MessageInputContext
 
 import SelectedPicker from '../common-content/contexts/attachment-picker-context/selected_picker.mdx';
 
-import AdditionalTextInputProps from '../common-content/core-components/channel/props/additional_text_input_props.mdx';
```

Comment:
We should always fetch them from its original source. So this change is not needed :)

Reply:
Fixed